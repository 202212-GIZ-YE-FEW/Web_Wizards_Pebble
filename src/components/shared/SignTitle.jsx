const SignTitle = ({ text }) => {
    return (
        <div className='flex justify-center lg:justify-start'>
            <h3 className='font-semibold mb-8 tracking-tight text-custom-blue lg:text-gray-800 text-3xl order-first'>
                {text}
            </h3>
        </div>
    );
};
export default SignTitle;
