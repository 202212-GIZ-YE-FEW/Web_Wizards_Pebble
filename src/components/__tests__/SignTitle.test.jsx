import renderer from "react-test-renderer";

import SignTitle from "@/components/shared/SignTitle";

it("renders correctly", () => {
    const tree = renderer.create(<SignTitle />).toJSON();
    expect(tree).toMatchSnapshot();
});
