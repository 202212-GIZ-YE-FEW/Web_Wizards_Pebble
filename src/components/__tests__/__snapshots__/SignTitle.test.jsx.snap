// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex justify-center lg:justify-start"
>
  <h3
    className="font-semibold !font-sans mb-8 tracking-tight text-custom-blue lg:text-gray-800 text-3xl order-first"
  />
</div>
`;
